examples/avmnist_multimodal/README.md

```diff
@@ -0,0 +1,40 @@
+# Multimodal Digit Classification: Multimodal Deep Learning (MMDL) Method on AVMNIST
+
+### 1. Description
+
+This example is constructed by refactoring the code of [MultiBench repository](https://github.com/pliang279/MultiBench) for [Pykale library](https://github.com/pykale/pykale).
+
+### 2. Usage
+* Dataset: [AVMNIST](https://drive.google.com/file/d/1KvKynJJca5tDtI5Mmp6CoRh9pQywH8Xp/view)
```

Comment:
Personal GDrive?

Reply:
This is the full dataset provided in the Multibench.

See https://github.com/pliang279/MultiBench#:~:text=MNIST%2C%20download%20the-,avmnist.tar.gz%20file,-from%20here%20and.